sugoi-api-core/src/main/java/fr/insee/sugoi/core/exceptions/AppCannotManagedAttributeException.java

```diff
@@ -0,0 +1,25 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package fr.insee.sugoi.core.exceptions;
+
+public class AppCannotManagedAttributeException extends RuntimeException {
```

Comment:
Not sure about that name, is it the app which cannot manage the attribute ?

Reply:
It's when a person (or a app account) cannot managed the attribute because he has not the good right